### PR TITLE
Added a new game

### LINKED ISSUE #4194: [New game]: Wordling
### 🎮 Game Request

Game logic and basic description

Components of the Game

Word List: A predefined list of acceptable words that can be used as the secret word and guessed words.

Secret Word: A randomly selected word from the word list that players need to guess.

Guess Input: Players input their guesses, which must be checked against the word list to ensure they are valid words.

Feedback Mechanism: After each guess, provide feedback to indicate which letters are correct and in the correct position, which are correct but in the wrong position, and which are incorrect.

Win/Lose Conditions: Determine when the player has won or lost the game based on the number of guesses.

### Point down the features

Game points

Scoring System

Base Points: Each guess earns a base number of points. For example, each guess could start with a base value of 10 points.

Bonus Points: Additional points are awarded for correct letters and correct positions.

Correct letter in the correct position (Green): +5 points.

Correct letter in the wrong position (Yellow): +2 points.

Penalty Points: Deductions can be applied for incorrect guesses or using too many attempts.

Incorrect letter: -1 point.

Exceeding a certain number of attempts could result in a penalty (optional).

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by codewithishu**
fixes #4194 
 

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

## Add your screenshots(Optional) 📸
Screenshot 2024-06-04 004406

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @codewithishu ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by codewithishu** [comment]
Can you merge it

File: assets/images/og-image (1).png
Comment (ishita-43) [inline assets/images/og-image (1).png:1]: Rename the images in assets folder according to the contribution guidelines

**Review by ishita-43 — CHANGES_REQUESTED** [review]
do same for all the images used in assests folder

File: assets/images/og-image (1).png
Comment (codewithishu) [reply assets/images/og-image (1).png:1]: Okay

**Comment by kunjgit** [comment]
Thank you  @codewithishu , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Comment by codewithishu** [comment]
> do same for all the images used in assests folder

I have done it

File: assets/images/og-image (1).png
Comment (codewithishu) [reply assets/images/og-image (1).png:1]: i have renamed it

**Comment by codewithishu** [comment]
@kunjgit please merge it

File: assets/images/og-image.png
Comment (ishita-43) [inline assets/images/og-image.png:1]: remove this if not needed

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: Games/Wordling/_config.yml
Comment (ishita-43) [inline Games/Wordling/_config.yml:1]: empty file remove this!!

File: Games/Wordling/dictionary.js
Comment (ishita-43) [inline Games/Wordling/dictionary.js:1]: also empty remove!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: Games/Wordling/index.js
Comment (ishita-43) [inline Games/Wordling/index.js:1]: empty !! remove this if not needed

File: Games/Wordling/package.json
Comment (ishita-43) [inline Games/Wordling/package.json:1]: Empty remove this if not needed!!

File: Games/Wordling/style.css
Comment (ishita-43) [inline Games/Wordling/style.css:1]: remove this if not needed

File: assets/images/og-image.png
Comment (codewithishu) [reply assets/images/og-image.png:1]: done

**Comment by ishita-43** [comment]
Please try to resolve merge conflicts

**Comment by codewithishu** [comment]
@kunjgit please merge it

**Review by ishita-43 — APPROVED** [review]
good to go!

**Comment by kunjgit** [comment]
Thank you  @codewithishu , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**